src/main/java/com/learntechfree/ltf/entity/Article.java

```diff
@@ -0,0 +1,79 @@
+package com.learntechfree.ltf.entity;
+
+import lombok.*;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+
+import javax.persistence.*;
+import java.sql.Timestamp;
+import java.util.Set;
+
+@Entity
+@Table(name = "articles",uniqueConstraints = {
+        @UniqueConstraint(columnNames = {"title","slug"})
+})
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
We don't need these.

Reply:
Actually had no idea what the are doing as well :D , will remove them :D

Reply:
They will create constructor as following, which we rarely require. We can use these annotation only incase we need. 

```java
class Demo
{
    private Long id;
    private String name;

     public Demo()
     {
         // no arg constructor 
     }

    // all args constructor
    public Demo(Long id, String name)
     {
          this.id  = id;
          this.name = name;
     }
}
```